test/unit_tests/modules/util/dom-traverse-spec.js

```diff
@@ -35,3 +35,87 @@ describe( 'Dom Traverse queryOne', function() {
     expect( query ).to.equal( null );
   } );
 } );
+
+describe( 'Dom Traverse getSiblings()', function() {
+  jsdom();
+
+  before( function() {
+    document.body.innerHTML =
+      '<div class="div-1"></div><div class="div-2"></div>';
+  } );
+
+  it( 'should return an array with a single sibling',
+    function() {
+      var elem = document.querySelector( '.div-1' );
+      var siblings = domTraverse.getSiblings( elem, 'div' );
+
+      expect( siblings.length ).to.equal( 1 );
+      expect( siblings[0].className ).to.equal( 'div-2' );
+    }
+  );
+} );
+
+describe( 'Dom Traverse not()', function() {
+  jsdom();
+
+  before( function() {
+    document.body.innerHTML =
+      '<div class="div-1"></div><div class="div-2"></div>';
+  } );
+
+  xit( 'should return an array with the item that wasn’t excluded',
```

Comment:
`.splice()` returns the removed items, so this fixes in the `not` function and your tests should pass. 

```javascript
function not( elems, exclude ) {
  var elemsArr = Array.prototype.slice.call( elems );
  var index = elemsArr.indexOf( exclude );
  if ( index > -1 ) {
    elemsArr.splice( index, 1 );
    return elemsArr;
  }
  return elemsArr;
}
```

Before I figured it out, I rewrote the test to make it a little more obvious what's going on.
```javascript
it( 'should return an array with the item that wasn’t excluded',
    function() {
      var items = document.querySelectorAll( 'div' );
      var exclude = document.querySelector( '.div-2' );

      items = domTraverse.not( items, exclude );

      expect( items.length ).to.equal( 1 );
      expect( items[0].className ).to.equal( 'div-1' );
    }
  );
```